viewer/web/daily.js

```diff
@@ -51,6 +51,9 @@ function update_ui(opinion) {
   // Set opinion title
   document.getElementById("opinion-h2").innerHTML = "Opinion for CSD #" + csd
 
+  // Set vote count
+  document.getElementById("opinion-count").innerHTML = "# of existing votes: " + opinion_count
```

Comment:
```suggestion
  document.getElementById("opinion-count").innerHTML = "# of existing votes: " + opinion[3]
```
You've already passed to the client all the vote counts in the `options` list.  You should use that, rather than trying to keep a separate variable updated.